File: 344-ReverseString/344-ReverseString.java
// Last updated: 7/31/2026, 8:28:27 PM
class Solution {
    public void reverseString(char[] s) {
        int i = 0;
        int j = s.length-1;

        while (j > i) {
            char t = s[i];
            s[i++] = s[j];
            s[j--] = t;
        }
    }
}

File: 344-ReverseString/344-ReverseString.cpp
// Last updated: 9/1/2026, 7:10:56 PM
1class Solution {
2public:
3    void reverseString(vector<char>& s) {
4        int i = 0;
5        int j = s.size() - 1;
6
7        while (i < j) {
8            char t = s[i];
9            s[i++] = s[j];
10            s[j--] = t;
11        }
12    }
13};